ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/general/">
+<chapter>
+<title>Maintaining your package</title>
+
+<section>
+<title>General responsibilities</title>
+<body>
+
+<note>
+Gentoo's Quality Assurance (QA) team also maintains a
+<uri link="https://projects.gentoo.org/qa/policy-guide/">policy guide</uri>,
+which contains useful advice on ebuild structure and behaviour.
+</note>
+  
+<p>
+You should always use the latest <uri link="::ebuild-writing/eapi/">EAPI</uri>.
+The main exception is when an eclass you consume does not yet support it.
+You should file a bug or otherwise contact the eclass maintainer to enquire
+about its status with respect to the new EAPI.
+
+<e>Rationale: Newer EAPIs add improvements which are often user-visible,
+e.g. EAPI 6's eapply_user and EAPI 7's improved cross-compile support.</e>
+</p>
+
+<p>
+Check regularly for new bugs on
+<uri link="https://bugs.gentoo.org/">bugs.gentoo.org</uri>.
+
+This includes CI/tinderbox style bugs, which are found by testing the latest
+ebuilds (by commit) and random packages, respectively.
+
+Note that <uri link="https://packages.gentoo.org/">packages.gentoo.org</uri>
+has an experimental Bugs listing per maintainer (see below) but it is
+not yet complete.
```

Comment:
I think this sentence is more distracting than it is helpful. Bugzilla has a powerful search function with the possibility to save searches, so why would anyone prefer p.g.o (which IIUC is aimed primarily at users) to Bugzilla itself?

Reply:
Agreed, let's restore it if the functionality is complete in b.g.o. Note that the maintainer pages on p.g.o are designed for developers and feedback is welcome on how to make them better for that.